### PR TITLE
DEM-2959 LiveConnect tracker module implementation

### PR DESCRIPTION
**Pull request by melgenek**

### CODE REVIEW

File: modules/liveConnect.js [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+import {hasGDPRConsent} from './userId/gdprUtils';
+import {detectPrng, factory} from 'ulid/dist/index';
+
+const prng = detectPrng(true);
+/** Generator of ULID identifiers. Falls back to insecure `Math.random` identifiers when `window.crypto` is not present in browser. */
+const ulid = factory(prng);
+
+const COOKIE = 'cookie';
+const LOCAL_STORAGE = 'html5';
+const LIVE_PIXEL_URL = '//rp.liadm.com/p';
```
Comment (jankoulaga) [inline modules/liveConnect.js]: I'd make this https always

File: integrationExamples/gpt/liveConnect_example.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
+<!--
+  This page calls a single bidder for a single ad slot. It can be considered a "hello world" example for using
```
Comment (jankoulaga) [inline integrationExamples/gpt/liveConnect_example.html]: Ehm.... this comment is wrong ;)

File: modules/consentManagement.js [hunk 3]
```diff
@@ -376,6 +376,7 @@ export function setConsentConfig(config) {
   }
   if (!addedConsentHook) {
     $$PREBID_GLOBAL$$.requestBids.before(requestBidsHook, 50);
+    if ($$PREBID_GLOBAL$$.liveConnectHook) $$PREBID_GLOBAL$$.liveConnectHook.before(requestBidsHook, 50);
```
Comment (bretg) [inline modules/consentManagement.js]: Please explain this -- why should the ConsentManagement module be related to LiveConnect?

File: modules/liveConnect.js [hunk 4]
```diff
@@ -0,0 +1,349 @@
+/**
+ * This module adds tracking support for LiveIntent's LiveConnect
+ * @module modules/liveConnect
```
Comment (bretg) [inline modules/liveConnect.js]: The naming convention is that these sub-modules must end in IdSystem. i.e. please change this file name to liveConnectIdSystem.js

File: integrationExamples/gpt/liveConnect_example.html (hunk 2 above)
Comment (melgenek) [reply integrationExamples/gpt/liveConnect_example.html]: 1

**Comment by melgenek** [comment]
@bretg 
`liveIntentIdSystem.js` - it it a submodule of `userId` module  that resolves identifiers based on cookie.
`liveConnect.js` (this pr) - it has to be a standalone module that drops our cookie and sends it to our pixel endpoint. We need to drop our own cookie to build some models on backend and be able to resolve identifiers in the `liveIntentIdSystem.js`

File: modules/consentManagement.js (hunk 3 above)
Comment (melgenek) [reply modules/consentManagement.js]: we want to setting our cookie if the consent is not given

**Pull request closed without merging**